core/src/test/scala/quasar/plugin/postgres/datasource/PostgresDatasourceSpec.scala

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2014–2019 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.plugin.postgres.datasource
+
+import slamdata.Predef._
+
+import argonaut._, Argonaut._
+import argonaut.JawnParser.facade
+
+import cats.~>
+import cats.effect._
+import cats.implicits._
+
+import doobie._
+import doobie.implicits._
+import doobie.util.ExecutionContexts
+
+import fs2.Stream
+
+import jawnfs2._
+
+import org.specs2.specification.BeforeAfterAll
+
+import quasar.ScalarStages
+import quasar.api.resource.{ResourcePathType => RPT, _}
+import quasar.connector.{ResourceError => RE, _}
+import quasar.contrib.scalaz.MonadError_
+import quasar.qscript.InterpretedRead
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import shims._
+
+object PostgresDatasourceSpec
+    extends DatasourceSpec[IO, Stream[IO, ?], RPT.Physical]
+    with BeforeAfterAll {
+
+  sequential
```

Comment:
Is this required?

Reply:
It is given the way I've constructed things, but I'll have a go at making the examples independent.

Reply:
@wemrysi having it `sequential` seems okay in that case. Could you add a comment to that effect here?

Reply:
Sure, I've also opened a ticket to remedy the situation, https://app.clubhouse.io/data/story/9544/improve-datasourcespec-by-representing-the-datasource-under-test-as-a-cats-effect-resource